roles/commcare_analytics/templates/superset/superset_config.py.j2

```diff
@@ -59,6 +59,7 @@ FEATURE_FLAGS = {
 
 # This is where async UCR imports are stored temporarily
 SHARED_DIR = '{{ superset_shared_dir }}'
+REMOVE_SHARED_FILES_AFTER = 7 # days
```

Comment:
Adds config for the following: https://github.com/dimagi/commcare-analytics/pull/90